ballerina/common_utils.bal

```diff
@@ -166,6 +166,15 @@ isolated function getOfType(__Type schemaType) returns __Type {
     }
 }
 
+isolated function getOfTypeName(__Type schemaType) returns string {
+    __Type? ofType = schemaType?.ofType;
+    if ofType is () {
+        return <string>schemaType.name;
+    } else {
+        return <string>getOfType(ofType).name;
+    }
+}
```

Comment:
Can't this be simplified like this?

```suggestion
isolated function getOfTypeName(__Type schemaType) returns string {
    return <string>getOfType(schemaType).name
}
```

We are using `getOfType()` function but here we are trying to do the first level of unwrapping inside this function, which is redundant.